docs/python/README.rst

```diff
@@ -0,0 +1,16 @@
+ONNXRuntime-Extensions
```

Comment:
this information is easily out of date, is this file needed?

Reply:
nope, removed.